deployments/nomad/otel-gateway.nomad

```diff
@@ -194,6 +194,10 @@ exporters:
     ingest_url: https://ingest.${SPLUNK_REALM}.signalfx.com
   debug:
     verbosity: detailed
+  otlphttp:
+    traces_endpoint: "https://ingest.${SPLUNK_REALM}.signalfx.com/v2/trace/otlp"
+    headers:
+      "X-SF-Token": "${SPLUNK_ACCESS_TOKEN}"
```

Comment:
Same here

Reply:
similar to agent-config.yaml, keeping sapm in the exporters but not in the pipeline

Reply:
Why do we need to keep it?

Reply:
Please remove sapm exporter here

Reply:
It's still there